src/less/segmented-buttons/stories/segmented-buttons.stories.js

```diff
@@ -0,0 +1,61 @@
+export default { title: "Segmented Buttons" };
```

Comment:
For stories, can we also please have examples of RTL, Font Size cascade, and color cascade (like we do in other places).